docs/Concepts/Consensus-Protocols/Comparing-PoA.md

```diff
@@ -4,9 +4,9 @@ description: Besu  Proof of Authority consensus protocols comparison
 
 # Comparing Proof of Authority consensus protocols
 
-Besu implements the Clique and IBFT 2.0 Proof of Authority consensus protocols. Proof of Authority
-consensus protocols work when participants know each other and there is a level of trust between
-them. For example, in a permissioned consortium network.
+Besu implements the Clique, IBFT 2.0, and QBFT Proof of Authority consensus protocols. Proof of
+Authority consensus protocols work when participants know each other and there is a level of trust
+between them. For example, in a permissioned consortium network.
 
 Proof of Authority consensus protocols have faster block times and a much greater transaction
 throughput than the Ethash Proof of Work consensus protocol used on the Ethereum MainNet.
```

Comment:
The paragraph below this should mention QBFT.

Reply:
Done